ui/apps/platform/src/Containers/Integrations/IntegrationTiles/BackupIntegrationsTab.tsx

```diff
@@ -0,0 +1,53 @@
+import type { ReactElement } from 'react';
+import { Alert, Gallery } from '@patternfly/react-core';
+
+import useFeatureFlags from 'hooks/useFeatureFlags';
+import useRestQuery from 'hooks/useRestQuery';
+import { fetchBackupIntegrations } from 'services/BackupIntegrationsService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+
+import {
+    backupIntegrationsDescriptors as descriptors,
+    backupIntegrationsSource as source,
+    getIntegrationsListPath,
+} from '../utils/integrationsList';
+import type { IntegrationsTabProps } from './IntegrationsTab.types';
+import IntegrationsTabPage from './IntegrationsTabPage';
+import IntegrationTile from './IntegrationTile';
+import { featureFlagDependencyFilterer, integrationTypeCounter } from './integrationTiles.utils';
+
+function BackupIntegrationsTab({ sourcesEnabled }: IntegrationsTabProps): ReactElement {
+    const { data, error } = useRestQuery(fetchBackupIntegrations);
```

Comment:
Certainly not specific to this PR, but it would be nice in a future pass to handle the loading state here as well. There might even be a pattern to reuse the loading/error/Gallery layout ala TBodyUnified here.

Reply:
Edit: aha I think you were referring to this here
>  Reusable component for common layout: IntegrationsTabPage

Reply:
Ah and here is the other piece :) 
> Integration source type tables: use table state and TbodyUnified element.